### ISSUE TITLE
Update contributing step by step

### ISSUE THREAD

**Issue by whisoer** [StratoDem/sd-material-ui#622]
## Description

I'm trying to contribute add a few components to this project, but I'm having a lot of errors following contributing step by step.

- `npm run build-dist ` actually doesn't exist, and following available scripts in package.json doesn't work
- `virtualenv venv -p python3` I try to resolve this dependency in KDE Neon/Ubuntu 20.04, but no success. I only could run it on Windows S.O.
- `npm run install-local` doesn't exist

**Comment by coralvanda** [comment]
Thank you for bringing this up. The documentation is a bit out of date. The command to run now is `npm run build:all`. 

You said that the scripts in `package.json` don't work. What was happening when you tried them? I am able to run the `build:all` command successfully.

What was the dependency issue you tried to resolve? I am able to set up the virtual environment in Ubuntu 20.04.

**Comment by whisoer** [comment]
Oh thank you for the quick answer! So I create a [select](https://material-ui.com/components/selects/) component and every time I try to run `npm run build:all` I have the following error:

![image](https://user-images.githubusercontent.com/33878228/111989668-d6797400-8af0-11eb-8184-ac84f0f30ebd.png)

I think it was an import error, but I'm not sure. I add imports following the tooltip PR changes (_imports_.py and index.js)

![image](https://user-images.githubusercontent.com/33878228/111990059-51db2580-8af1-11eb-8774-f756c5d989ab.png)

I'm not sure if I need to run something before `npm i` and `npm run build:all`, I try using `npm ci` too. So if you could help me, I really want to contribute to this project bring more components.

About ubuntu, could you please share `virtualenv` and `python` versions are you using?

How I'm having problems in all steps I'm not sure what exactly dependency I'm trying to resolve, maybe resolving `npm run build:all` all work. And after running these, I can test it locally using `npm start`?

**Comment by coralvanda** [comment]
You shouldn't need to add any imports manually in the `sd_material_ui/__imports__.py` file. It does look like there was a missing import in the `index.js` file. I see from the image you attached there is a change there. Does this same error occur even after adding that import?

I'm using `virtualenv 20.0.17` and `Python 3.8.5` for this project at the moment.

**Comment by whisoer** [comment]
Oh, I got it! Thank you for sharing.

And I add select to index as following image, but even removing manually import from `sd_material_ui/__imports__.py ` file, same error running `npm run build:all`.

![image](https://user-images.githubusercontent.com/33878228/111992514-46d5c480-8af4-11eb-8173-1f507b9f68dd.png)

And with the same versions, I try to run it from the beginning. Deleted `node_modules` and running `npm install`  I got this error that I probably missed, not 

![image](https://user-images.githubusercontent.com/33878228/111994186-3cb4c580-8af6-11eb-89ef-5240a1ce379a.png)
